tuberculosis/lib/pages/information_tab_page.dart

```diff
@@ -96,38 +96,56 @@ class VideoScreen extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    List<String> sampleData = <String>[
-      "https://www.youtube.com/watch?v=yR51KVF4OX0",
-      "https://www.youtube.com/watch?v=yR51KVF4OX0"
-    ];
-    List<Question> questions = <Question>[
-      const Question("Hoi"),
-      const Question("Vraag 2"),
-      new Question("ASdadasda", answers: ["asdsadsadasadsasaasdas", "bgbgbgbgbgbg", "cdiasdihsjsdjfhskjfhskdfhkshjkjkhksfdhjkh"], correctAnswer: 2)
-    ];
+    List<Widget> children = List<Widget>();
+
+    children.add(new GestureDetector(
+      onTap: () => _openVideo(_video.reference),
+      child: new Image.network(
+          "http://img.youtube.com/vi/" +
+              getIdFromUrl(_video.reference) +
+              "/hqdefault.jpg",
+          fit: BoxFit.cover),
+    ));
+
+    if (_video.quiz != null) {
+      children.add(new QuizWidget(_video.quiz.questions));
+    }
+
     return new CupertinoPageScaffold(
         navigationBar: new CupertinoNavigationBar(
           middle: new Text(_video.title),
         ),
         child: new Material(
             child: new SafeArea(
                 child: new Column(
-                  children: [
-                    new GestureDetector(
-                      onTap: () => _openVideo(_video.reference),
-                      child: new Image.network(
-                          "http://img.youtube.com/vi/" +
-                              getIdFromUrl(_video.reference) +
-                              "/hqdefault.jpg",
-                          fit: BoxFit.cover),
-                    ),
-                    new QuizWidget(questions)
-                  ],
+                  children: children,
                 ))));
   }
 
   String getIdFromUrl(String url) {
```

Comment:
R O B U U S T

Je zou kunnen beargumenteren dat deze code niet in de app thuishoort maar in de webapp, maar dat terzijde